modules/distr-tracing-tempo-install-tempomonolithic-cli.adoc

```diff
@@ -0,0 +1,115 @@
+// Module included in the following assemblies:
+//
+//* observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="distr-tracing-tempo-install-tempomonolithic-cli_{context}"]
+= Installing a TempoMonolithic instance by using the CLI
+
+:FeatureName: The TempoMonolithic instance
+include::snippets/technology-preview.adoc[]
+
+You can install a TempoMonolithic instance from the command line.
+
+.Prerequisites
+
+* An active {oc-first} session by a cluster administrator with the `cluster-admin` role.
++
+[TIP]
+====
+* Ensure that your {oc-first} version is up to date and matches your {product-title} version.
+
+* Run `oc login`:
++
+[source,terminal]
+----
+$ oc login --username=<your_username>
+----
+====
+
+.Procedure
+
+. Run the following command to create a project of your choice for the TempoMonolithic instance that you will create in a subsequent step:
++
+[source,terminal]
+----
+$ oc apply -f - << EOF
+apiVersion: project.openshift.io/v1
+kind: Project
+metadata:
+  name: <project_of_tempomonolithic_instance>
+EOF
+----
+
+. Decide which type of supported storage you will use for storing traces: in-memory storage, a Persistent Volume, or object storage.
++
+[IMPORTANT]
+====
+Object storage is not included with the {TempoShortName} and requires setting up an object store by a supported provider: link:https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation[Red Hat OpenShift Data Foundation], link:https://min.io/[MinIO], link:https://aws.amazon.com/s3/[Amazon S3], link:https://azure.microsoft.com/en-us/products/storage/blobs/[Azure Blob Storage], or link:https://cloud.google.com/storage/[Google Cloud Storage].
+
+Additionally, opting for object storage requires creating a secret for your object storage bucket in the project that you created for the TempoMonolithic instance. You can do this by running the following command:
```

Comment:
```suggestion
Additionally, opting for object storage requires creating a secret for your object storage bucket in the project that you created for the `TempoMonolithic` instance. You can do this by running the following command:
```

Reply:
I wouldn't add back ticks to every mention of the TempoStack product. I only use back ticks when referring to the custom resource and it's contents. It's like because it's possible to open Firefox by running the `firefox` command, should we write `firefox` rather than "Firefox" everywhere? IMO, the style guide rules should be friends with the common sense.

Reply:
> In the intro, you use backticks.

True, but for the custom resource, not the instance.